README.md

```diff
@@ -83,7 +83,29 @@ $ nodemon bin/www
 ```shell
   http://localhost:3000/
 ```
+### Docker setup
 
+> Docker containerization 
+`docker build -t my-app --platform linux/amd64 .` 
+
+> Docker run
+`docker run -p 3000:3000 my-app`
+
+> **Warning**
+> You'll need your Mongo DB to connect to the app and
+> you'll need to update the Mongo URI to connect to your Mongo DB.
```

Comment:
It's not clear to me what I actually need to have setup with MongoDB before running docker run on this container.

Looking at `docker-compose.yml`, it seems like that just starts a basic mongodb container?

Reply:
Yes its just to start basic mongodb container